src/libutil/experimental-features.cc

```diff
@@ -294,6 +294,14 @@ constexpr std::array<ExperimentalFeatureDetails, numXpFeatures> xpFeatureDetails
         )",
         .trackingUrl = "https://github.com/NixOS/nix/milestone/48",
     },
+    {
+        .tag = Xp::PipeOperators,
+        .name = "pipe-operators",
+        .description = R"(
+            Enables |> and <| operators.
+        )",
+        .trackingUrl = "https://github.com/NixOS/rfcs/pull/148",
```

Comment:
This should presumably be a milestone instead, and if someone creates one I will be happy to stick it here.

Reply:
```suggestion
        .trackingUrl = "https://github.com/NixOS/nix/milestone/55",
```